doc/rst/technotes/reduceIntents.rst

```diff
@@ -73,7 +81,7 @@ Increment ``x`` in the loop -- counts the number of iterations:
 
  .. code-block:: chapel
 
-  var x = 5;
+  var x = 0;
   forall myIterator() with (+ reduce x) {
     x += 1;
   }
```

Comment:
The writeln() that follows this loop no longer seems accurate due to the non-zero initial value of 'x'.

Reply:
I changed the initial value for 'x' from 5 to 0, so this should be correct now. ?

Reply:
Oh wait, I read the diff backwards.  Nevermind.